Update 37 Templates.md (#1400)
DevExpress/devextreme-documentation: concepts/55 Vue Components/20 Component Configuration Syntax/37 Templates.md

    }
    </script>

> [important] In Vue 2 applications, a`<template>` should not contain more than one root element. Vue 3 applications has no such restriction.

Reviewer: ```suggestion
[important] In Vue 2 applications, a`<template>` should not contain more than one root element. Vue 3 applications have no restriction.
```
